pallets/staking-rewards/src/lib.rs

```diff
@@ -467,6 +469,36 @@ impl<T: Config> OnUnbalanced<NegativeImbalanceOf<T>> for OffsetPool<T> {
     }
 }
 
+// DustRemoval handler
```

Comment:
what was the problem of previous impl? trait bounds? than add todo to remove initial one

Reply:
Conflicting implementation. Since `balances::DustHandler` switched to `[Credit](fungible::Credit)` from `Imbalance`. But we are using `Imbalance` in `fn align_supply`

Reply:
Yeah, add todo pls

Reply:
We don't need Todo. Both structs are for two different implementation and purposes